drivers/include/periph/usbdev.h

```diff
@@ -158,6 +158,16 @@ typedef enum {
      */
     USBDEV_EVENT_SOF,
 
+    /**
+     * @brief USB suspend condition active
+     */
+    USBDEV_EVENT_SUSPEND,
+
+    /**
+     * @brief USB suspend condition no longer active
+     */
+    USBDEV_EVENT_RESUME,
```

Comment:
dump question here:
Is this suppose to be random or alphabetical order ?

Reply:
Good question. The only "order" I had in mind was something that appeared logical to me. Which at the moment is to group the similar events together as much as possible.

Reply:
Alright, I don't think the coding guideline force us to change something here. Let's keep it this way.